### ISSUE TITLE
Add gridstatus to conda-forge channel

### ISSUE THREAD

**Issue by gsheni** [kmax12/gridstatus#145]
- The instructions here (https://conda-forge.org/docs/maintainer/adding_pkgs.html) can be followed for adding packages to the conda-forge channel and allowing conda users to install the library.

**Comment by kmax12** [comment]
can't conda user install the library with pip inside of their conda env?

**Comment by gsheni** [comment]
Yep that's fine. If other users like this issue and/or comment, then it can be revisited

**Comment by kmax12** [comment]
I agree. will watch for comments!

**Comment by rsmith54** [comment]
I'd really appreciate seeing this added to conda.

pip can be used with conda, but the chances of conflicts and dependency issues increase dramatically when mixing the two.
